lxcfs.c

```diff
@@ -82,22 +186,27 @@ static void do_reload(void)
 		goto good;
 
 	dlopen_handle = dlopen("/usr/lib/lxcfs/liblxcfs.so", RTLD_LAZY);
```

Comment:
Initial impression - looks great, thanks much.  I had a few comments last night, will post them later.  (It's possible I'll answer my own questions on a second read-through).